public/cli/lib/rendering/render.html

```diff
@@ -134,7 +139,12 @@
               next: "p" + (i + 1),
             });
             // concat the lines together with all words inside
-            lyrics += "<div class='phrase' id='p" + i + "'>" + text + "</div>";
+            lyrics += "<div class='phrase";
```

Comment:
This is not related to your work on the headers but I notice that the single and double quotes are the wrong way around. Our code should produce the `class` attribute using double quotes rather than single quotes for its value.